core/src/ibc/channel_04/manager.rs

```diff
@@ -15,7 +15,9 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
What do you mean by the commit message "Refactor ICS data that are stored in hash"?
Is "in hash" commonly used term?

Reply:
`in hash` was meant to be `as a hash`. @ScarletBlue answered that `as a hash` would be more natural. I'll update it